README.md

```diff
@@ -7,87 +7,115 @@ This repository's `master` branch tracks images pushed to the official Demisto D
+You can contribute in the following ways:
 
-When opening a new pull request, a bot will evaluate whether you have signed the CLA. If required, the bot will comment on the pull request, including a link to accept the agreement. The CLA document is also available for review as a [PDF](https://github.com/demisto/content/blob/master/docs/cla.pdf).
+* [**Create a new Docker Image**](#create-a-new-docker-image) for use in an [XSOAR/XSIAM integration or script](https://xsoar.pan.dev/docs/integrations/docker#scriptintegration-configuration).
+
+* **Update existing Docker Images:** to fix issues or security concerns in our `Dockerfiles`. 
+
+* **Enhancements:**  such as optimizations, helper scripts, or improved configurations.
+
+* **Documentation:** Documentation is crucial. If you find gaps or errors in our documentation, please help us improve it. You can suggest clarifications or additions to make it more user-friendly.
+
+### CLA Licenses
+All contributors are required to sign a contributor license agreement (CLA) to ensure that the community is free to use your contributions.
+
+When opening a new pull request, a bot will evaluate whether you have signed the CLA. If required, the bot will comment on the Pull Request, including a link to accept the agreement. The CLA document is also available for review as a [PDF](https://github.com/demisto/content/blob/master/docs/cla.pdf).
 Visit our [Frequently Asked Questions](https://xsoar.pan.dev/docs/concepts/faq#cla-is-pending-even-though-i-signed-the-agreement) article for CLA related issues.
 
-After opening your docker pull request, and in order for the reviewer to understand the context, make sure to link to the corresponding pull request from the [Content](https://github.com/demisto/content) repo where this docker image will be used.
-## Getting Started
-Each docker image is managed in its own directory. The directory should be named the same as the image name (without the organization prefix). If needed, we prefer using a dash (`-`) as a separator in the name. All image directories are located under the `docker` dir.
+After opening a Pull Request, and in order for the reviewer to understand the context, make sure to link to the corresponding Pull Request from the [Content](https://github.com/demisto/content) repo where this Docker image will be used.
 
-The directory should contain one Dockerfile which will be used for building the docker image. Each image when it is built is tagged with the commit hash and version. 
 
-The script `docker/build_docker.sh` is used to build all modified docker images. The script detects modified directories by comparing against origin/master if on a branch or if on master by using the CIRCLE_COMPARE_URL environment variable to obtain the commit range of the current build.
+### How to Contribute
```

Comment:
why have contributing and how to contribute as 2 seperate sections?

Reply:
Addressed in https://github.com/demisto/dockerfiles/pull/20820/commits/5d530e9e0adcd77c6df52a7cdf1fe3692d1f8d37.